etcdctlv3/command/get_command.go

```diff
@@ -69,6 +70,14 @@ func getGetOp(cmd *cobra.Command, args []string) (string, []clientv3.OpOption) {
 	}
 
 	opts := []clientv3.OpOption{}
+	switch getConsistency {
+	case "s":
+		opts = append(opts, clientv3.WithSerializable())
+	case "l":
+	default:
+		ExitWithError(ExitError, fmt.Errorf("unknown consistency flag %q", getConsistency))
```

Comment:
@heyitsanthony should this be https://github.com/coreos/etcd/blob/master/etcdctlv3/command/error.go#L30?